lib/codecs/src/gelf.rs

```diff
@@ -69,3 +69,7 @@ pub(crate) static GELF_TARGET_PATHS: Lazy<GelfTargetPaths> = Lazy::new(|| GelfTa
 /// Additional field names must also be prefixed with an `_` , however that is intentionally
 /// omitted from this regex to be checked separately to create a specific error message.
 pub static VALID_FIELD_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r"^[\w\.\-]*$").unwrap());
+
+/// Regex for matching valid field names in the decoder. Additionally, allow @ characters
+/// produced by Graylog GELF outputs, which are strictly not valid in the GELF specification
```

Comment:
**🥜 nitpick:** It'd also be nice to mention here the "Output" feature, since I couldn't find anywhere in the graylog docs examples of the "@" character, for future readers of the code.

<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
‏

```suggestion
/// produced by Graylog GELF outputs, which are strictly not valid in the GELF specification.
```